### PR TITLE
Behringer Crave

### PR DESCRIPTION
**Pull request by ptah23**
Everything is connected up, EXCEPT:
1 knob and 1 toggle does not work for me in index.html when I toggle, or turn them. I'm not sure what the problem is

### CODE REVIEW

File: js/behringer-crave.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+const TOGGLES = [
+  {idx: 0, x: 300, y: 375, type: "twoway", name: "VCO WAVE"},
+  {idx: 1, x: 473, y: 375, type: "twoway", name: "VCO MOD SOURCE"},
+  {idx: 2, x: 645, y: 375, type: "twoway", name: "VCO MOD DEST"},
+  {idx: 3, x: 998, y: 475, type: "twoway", name: "LFO WAVE"},
+  {idx: 4, x: 825, y: 375, type: "twoway", name: "VCF MODE"},
+  {idx: 5, x: 998, y: 375, type: "twoway", name: "VCF MOD SOURCE"},
+  {idx: 6, x: 645, y: 475, type: "twoway", name: "SUSTAIN"},
+  {idx: 7, x: 1344, y: 375, type: "twoway", name: "VCA MODE"},
+  {idx: 8, x: 1980, y: 375, type: "twoway", name: "VCF MOD POLARITY"}
```
Comment (zwippie) [inline js/behringer-crave.js]: `{idx: 8, x: 1168, y: 375, type: "twoway", name: "VCF MOD POLARITY"}`

File: js/behringer-crave.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      knobs: KNOBS,
+      knobTypes: KNOB_TYPES,
+      toggles: TOGGLES,
+      toggleOptions: {
+        types: TOGGLE_TYPES,
+        width: TOGGLE_WIDTH,
+        height: TOGGLE_HEIGHT
+      },
+      initialValues: {
+        knobValues: Array(14).fill(150),
```
Comment (zwippie) [inline js/behringer-crave.js]: `knobValues: Array(15).fill(158),`

Centers all the knobs on an initial patch and fixes the broken tempo knob.

File: js/behringer-crave.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    //outputs
+    //row 2
+    {idx: 25, col: 9, row: 1, x: 915, y: 175, name: "LFO TRI", input: false},
+    {idx: 26, col: 10, row: 1, x: 980, y: 175, name: "LFO SQ", input: false},
+    {idx: 27, col: 11, row: 1, x: 1045, y: 175, name: "VC MIX", input: false},
+    {idx: 28, col: 12, row: 1, x: 1110, y: 175, name: "ASSIGN", input: false},
+    {idx: 29, col: 13, row: 1, x: 1177, y: 175, name: "KB", input: false},
+    {idx: 30, col: 14, row: 1, x: 1247, y: 175, name: "GATE", input: false},
+    {idx: 31, col: 15, row: 1, x: 1310, y: 175, name: "VCF", input: false},
+]
```
Comment (zwippie) [inline js/behringer-crave.js:53]: Add the phones output too. Could be patched to ext input for some *overdrive* (like on the Model D).

**Review by zwippie — COMMENTED** [review]
Looking great! Just some small things that need to be fixed and it's ready to go.

About the images:
1. The file size of the front panel image is a bit too large (1.66MB). Try to save it as a lower quality (8-bit?) png so it's about 500KB.
2. The switch-right image should be flipped, not rotated from the original. Now the shadow is a bit off in the right position.

File: js/behringer-crave.js (hunk 1 above)
Comment (ptah23) [reply js/behringer-crave.js]: fixed

File: js/behringer-crave.js (hunk 3 above)
Comment (ptah23) [reply js/behringer-crave.js:53]: added, i should try doing this irl ;)

**Review by ptah23 — COMMENTED** [review]
fixes made

File: js/behringer-crave.js (hunk 1 above)
Comment (ptah23) [reply js/behringer-crave.js]: fixed

**Review by zwippie — APPROVED** [review]
Excellent!

**Pull request merged**